lib/pages/home/home_page.dart

```diff
@@ -203,3 +185,111 @@ class ProfileCard extends StatelessWidget {
     );
   }
 }
+
+class FavouriteWebsitesCard extends StatelessWidget {
```

Comment:
Why did you add this back? Looks like a messed up merge. I generified the card and deleted this class.